task_1.py

```diff
@@ -0,0 +1,12 @@
+import json
+from typing import Dict
+
+
+def create_full_name() -> Dict:
```

Comment:
I think it is better if the record loading and the record transformation is kept separate (Single Responsibility Principle).